Update new developer readme (#392)
hackforla/VRMS: CONTRIBUTING.md

## How to contribute

> If you are interested in joining the project, then please review our [Prroject Wiki](https://github.com/hackforla/VRMS/wiki/Join-the-Team) and then review our [Project Board](https://github.com/hackforla/VRMS/projects/2) to see if anything interests you.

Reviewer: ```suggestion
If you are interested in joining the project, then please review our [Project Wiki](https://github.com/hackforla/VRMS/wiki/Join-the-Team) and then review our [Project Board](https://github.com/hackforla/VRMS/projects/2) to see if anything interests you.
```

### Committing changes overview

> 1. Find a ticket in the [Prioritized Backlog](https://github.com/hackforla/VRMS/projects/2) and assign yourself to the ticket, and update its status as "In progress".

Reviewer: ```suggestion
1. Find a ticket in the [Prioritized Backlog](https://github.com/hackforla/VRMS/projects/2), assign yourself to the ticket, and move the ticket to the "In progress" column.
```

1. Fork the repo.

1. Make a branch based on the `development` branch.

> 1. Develop the task solution.

Reviewer: if "task solution" is common jargon among in software dev teams, fine. If not, I think this can be reworded to be clearer.

1. Commit and push your changes to your forked repository. Please pull the recent changes from `vrms/development` before pushing your changes.

1. From the `VRMS` repository, create a Pull Request from your branch into the VRMS `developnent` branch.

1. Any changes will need approval and merging from a tech lead.

> 1. Bring an update of your changes to the weekly VRMS team checkin.

Reviewer: does this refer to discussing the update at the monday meeting, updating the github issue, or both? Once that's clear, I will propose reworded text

### Forking and cloning the repository

#### Step 1: Become a member of the repository Team

> In the `vrms` slack channel, send your GitHub name to the project manager (or on the slack channel thread) and we'll add you as a member to the GitHub repository Team.

Reviewer: ```suggestion
In the `vrms` slack channel, share your GitHub name so a project manager can add you to the HackforLA GitHub organization and the VRMS repository team.
```

Reviewer: 1. did i get the copywriting right? we want to have a HfLA admin add the user to the github org, and then we'll also add them to the VRMS "write" team on GH, too?
2. also, we can link straight to the slack channel from here, right?

Once you have accepted the GitHub invite (via email or in your GitHub notifications), please do the following:

1. Mark your own membership public https://help.github.com/en/articles/publicizing-or-hiding-organization-membership#changing-the-visibility-of-your-organization-membership

> 1. Setup two factor authentication on your account https://github.com/hackforla/governance/issues/20

Reviewer: ```suggestion
1. Setup two-factor authentication on your account https://github.com/hackforla/governance/issues/20
```

> These steps are manditory in order to contribute to all HackforLA projects.

Reviewer: ```suggestion
These steps are mandatory in order to contribute to any HackforLA project.
```

## How to contribute

If you are interested in joining the project, then please review our [Projects Board](https://github.com/hackforla/VRMS/projects/2) to see if anything interests you. If so, then please do the following.

1. Review the Hack for LA Getting Started Guide https://www.hackforla.org/getting-started.

1. Join the VRMS weekly meetings https://www.vrms.io/.

> 1. Reach out to our project leads to discuss how you would like to contribute.

Reviewer 2: I think it should be the same or almost the same as the first paragraph of https://github.com/hackforla/VRMS/wiki/Join-the-Team 
The wiki page was updated by Adam recently.

Author: 👍

### Committing changes overview

> 1. Find a ticket in the [Prioritized Backlog](https://github.com/hackforla/VRMS/projects/2) and assign yourself to the ticket.

Reviewer 2: and assign yourself to the ticket. => , assign yourself to the ticket, and update its status as "In progress".

Author: 👍

1. Fork the repo.

> 1. Make a branch off of `development`.

Reviewer 2: Make a branch off of `development`. => Make a new branch based on the `development` branch.

Author: 👍

> 

Reviewer 2: Please add the following item:

1. Develop the task solution.

Author: 👍

> 1. Commit and push your changes to your forked repository.

Reviewer 3: It will be great to add that need to pull changes before push them to the repo.

Commit, pull updates from `development` branch and push your changes to your forked repository.

Author: 👍

1. From the `VRMS` repository, create a Pull Request from your branch into the VRMS `developnent` branch.

> 1. Any changes will need approval and merging from a project leads.

Reviewer 3: @nickbeaird 
Currently, we have positions: tech lead and product manager. What do you mean about 'project leads'?

Author: 👍

> 1. Any changes will need approval and merging from a project leads.

Reviewer 2: from a project leads => from project leads

Author: 👍

1. Bring an update of your changes to the weekly VRMS team checkin.

> ### Forking and cloning the repository with proper security

Reviewer 2: The header name is confusing for me. Do we allow contributing without security?
I think we should not allow non-members to contribute to the project.

Author: 👍

> #### Step 1: Become a member of the repository Team

Reviewer 2: It intersects with my first comment about joining the project. So it is covered by https://github.com/hackforla/VRMS/wiki/Join-the-Team 
I mean that contributing to the project and joining the team is the same.

Author: 👍
